PhysicsTools/PatAlgos/python/producersLayer1/jetUpdater_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-patJetsUpdated = cms.EDProducer("PATJetUpdater",
+updatedPatJets = cms.EDProducer("PATJetUpdater",
```

Comment:
@slava77 well, the patJetsUpdated module was actually the standard recipe in JEC twiki page for 763:
https://twiki.cern.ch/twiki/bin/view/CMSPublic/WorkBookJetEnergyCorrections?rev=132#CorrPatJets